concepticondata/conceptlists/Goz-2017-600.tsv

```diff
@@ -0,0 +1,601 @@
+Goz-2017-600-119	119	dert	trouble			3.27	2.41	3.6	3.83	3.29	3.44	2.16	158	1	4
+Goz-2017-600-120	120	devlet	state	2955	STATE (POLITICS)	3.11	2.57	3.39	3.51	2.99	3.73	3.73	902	2	6
+Goz-2017-600-121	121	deyim	idiom			3.38	2.67	3.23	4.09	3.51	3.11	2.84	60	2	5
+Goz-2017-600-122	122	dikiş	seam			2.18	1.97	2.51	2.21	2.06	5.52	6.33	24	2	5
+Goz-2017-600-123	123	din	religion	661	RELIGION	2.22	2.01	2.4	2.62	1.91	3.38	2.22	261	1	3
+Goz-2017-600-124	124	dinamik	dynamic			4.26	2.71	4.73	4.81	4.77	3	2.76	81	3	7
+Goz-2017-600-125	125	dizgin	reins			3.87	2.7	4.9	4.37	3.49	3.42	5.31	8	2	6
+Goz-2017-600-126	126	dostluk	friendship			2.57	1.95	2.83	2.84	2.67	4.07	1.97	86	2	7
+Goz-2017-600-127	127	dram	drama			4.15	3.04	4.34	4.76	4.42	2.82	2.84	50	1	4
+Goz-2017-600-128	128	dua	prayer			1.82	1.75	1.9	1.97	1.67	4	2.84	51	2	3
+Goz-2017-600-129	129	dudak	lip	478	LIP	1.71	1.63	1.61	1.89	1.71	6.46	6.8	157	2	5
+Goz-2017-600-130	130	duygulu	sentimental			2.99	2.03	3.3	3.58	3.09	3.44	2.02	16	3	7
+Goz-2017-600-131	131	dükkân	shop	746	SHOP	1.89	1.76	1.94	2.09	1.81	6.17	6.73	205	2	6
+Goz-2017-600-132	132	dürüst	honest	1120	HONEST	2.84	2.06	3	3.55	2.8	3.3	2.09	54	2	6
+Goz-2017-600-133	133	ebleh	imbecile			5.71	3.3	6.06	6	5.55	1.85	2.67	2	2	5
+Goz-2017-600-134	134	ekonomi	economy			4.33	2.87	3.89	4.97	5.37	3.21	3.47	336	4	7
+Goz-2017-600-135	135	ekran	screen			3.4	2	2.38	3.45	5.43	6.27	6.74	144	2	5
+Goz-2017-600-136	136	elektrik	electricity	59	ELECTRICITY	2.3	1.99	2.01	2.56	2.57	4.57	5.31	260	4	8
+Goz-2017-600-137	137	elem	sorrow	1668	GRIEF	4.44	2.82	5.56	4.57	3.92	2.32	1.78	11	2	4
+Goz-2017-600-138	138	eleştirmen	critic			4.68	3.06	4.41	5.51	2.5	3.8	5.34	22	4	10
+Goz-2017-600-139	139	elma	apple	1320	APPLE	1.52	1.51	1.43	1.53	1.59	6.73	6.95	64	2	4
+Goz-2017-600-140	140	enerjik	energetic			3.95	2.86	3.45	4.55	4.79	3.65	2.69	14	3	7
+Goz-2017-600-141	141	engel	obstacle			2.75	2.08	3.18	3.19	2.63	3.97	4.42	126	2	5
+Goz-2017-600-142	142	enginar	artichoke			3.92	2.76	3.84	4.85	4.14	5.59	6.74	31	3	7
+Goz-2017-600-143	143	erozyon	erosion			4.09	2.99	4.43	4.17	2.76	4.26	6.01	21	3	7
+Goz-2017-600-144	144	esinti	breeze			2.89	2.27	3.22	3.19	2.92	3.99	5.36	17	3	6
+Goz-2017-600-145	145	esir	captive			3.14	2.32	3.73	3.57	3.02	4.64	5.06	30	2	4
+Goz-2017-600-146	146	espri	joke	3425	JOKE	3.61	2.4	3.42	4.15	4.35	3.61	2.76	28	2	5
+Goz-2017-600-147	147	eşek	donkey	1862	DONKEY	1.59	1.52	1.66	1.72	1.49	6.4	6.8	52	2	4
+Goz-2017-600-148	148	eşit	equal	1570	EQUAL	2.72	2.05	2.96	3.17	2.72	3.57	2.78	103	2	4
+Goz-2017-600-149	149	eşsiz	unique			3.7	2.48	3.86	4.5	3.95	2.16	2.35	36	2	5
+Goz-2017-600-150	150	eşya	item	2981	ITEM	1.7	1.47	1.67	1.85	1.79	5.15	6.56	199	2	4
+Goz-2017-600-151	151	et	meat	634	MEAT	1.67	1.5	1.75	1.9	1.56	6.43	6.76	259	1	2
+Goz-2017-600-152	152	etken	factor			4.3	3.01	4.86	4.91	4.39	2.57	2.98	104	2	5
+Goz-2017-600-153	153	ev	house	1252	HOUSE	1.45	1.37	1.41	1.51	1.49	6.54	6.69	1887	1	2
+Goz-2017-600-154	154	evlât	offspring			2.34	2.03	2.71	2.55	2.15	5.29	6	78	2	5
```

Comment:
I wasn't sure about this because there is another term translated with "offspring", so it could also be a distinction between male versus female offspring.  I'd leave it unmapped for now.